backend/review/models.py

```diff
@@ -94,6 +94,18 @@ def get_averages(topic_id, instructor_name=None, fields=None):
 ALL_FIELD_SLUGS = [x[2] for x in REVIEW_BIT_LABEL]
 
 
+class CachedReviewResponse(models.Model):
+    """
+    Represents a mapping from temp_topic_id (string-delimited list of sorted courses within a topic)
+    to a JSON object storing summarized course review data (all the data frontend uses to display
+    reviews).
+    """
+
```

Comment:
I think topic_id should be unique=True here (theoretically our loading script should enforce this, but I think its good to specify here too?)